### PR TITLE
Don't instantiate unused analytics client

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* Don't instantiate the GA client, as it's not used anywhere. The scope of https://github.com/studiopress/genesis-custom-blocks/pull/101 didn't include using the analytics.
* The client also uses a `GA_ID` that's copied from Genesis Blocks. I need to fix that.
* Update the Copyright year to 2022

#### Testing instructions
* Just a sanity-check would be great

### CODE REVIEW

File: js/src/block-editor/index.js [hunk 1]
```diff
@@ -11,15 +11,9 @@ import { setLocaleData } from '@wordpress/i18n';
...
 
 setLocaleData( { '': {} }, 'genesis-custom-blocks' );
 addFilter( 'genesisCustomBlocks.controls', 'genesisCustomBlocks/addControls', addControls );
 
 // @ts-ignore
 registerBlocks( genesisCustomBlocks, gcbBlocks, Edit );
-
-// @ts-ignore
-window.GcbAnalytics = {
-	GAClient: new GAClient(),
```
Comment (kienstra) [inline js/src/block-editor/index.js:24]: This isn't used anywhere, at least yet. 

And I mistakenly kept in the copied `GA_ID` from Genesis Blocks.

File: bin/tag-built.sh [hunk 2]
```diff
@@ -7,14 +7,14 @@ npm ci
...
+	echo "Error: Unable to determine tag."
+	exit 1
 fi
 
 built_tag="$tag-built"
 if git rev-parse "$built_tag" >/dev/null 2>&1; then
-    echo "Error: Built tag already exists: $built_tag"
-    exit 2
+	echo "Error: Built tag already exists: $built_tag"
+	exit 2
```
Comment (kienstra) [inline bin/tag-built.sh:17]: Just changes spaces to tabs to match other file

**Comment by kienstra** [comment]
Hi @johnstonphilip,
Would you have time to review this? Thanks a lot!

**Review by dreamwhisper — APPROVED** [review]

**Comment by kienstra** [comment]
@dreamwhisper, thanks a lot for reviewing this, and on your birthday! 🎂

**Comment by johnstonphilip** [comment]
Code looks straightforward and fine - smoke test showed no problems that I could see.

**Review by johnstonphilip — APPROVED** [review]

**Comment by kienstra** [comment]
@johnstonphilip, thanks a lot for testing this and approving it!

**Pull request merged**